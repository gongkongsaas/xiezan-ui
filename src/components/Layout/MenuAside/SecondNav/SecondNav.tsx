import { defineComponent } from 'vue'

export default defineComponent({
  name: 'SecondNav',
  setup(props, { slots }) {
    return () => (
      <div class="second-nav">{slots.default && slots.default()}</div>
    )
  },
})
