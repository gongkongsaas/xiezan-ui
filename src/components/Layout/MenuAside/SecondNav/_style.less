.second-nav {
    width: 130px;
    height: calc(100vh - 50px);
    padding-top: 8px;
    padding-bottom: 40px;
    background-color: #ffffff;
    color: rgba(212, 212, 212, 0.996);
    box-sizing: border-box;
    overflow-x: hidden;
    overflow-y: auto;
    scrollbar-width: none;
}