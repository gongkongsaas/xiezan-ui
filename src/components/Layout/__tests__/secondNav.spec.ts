import { shallowMount } from '@vue/test-utils'
import SecondNav from '../menu-aside/second-nav/secondNav'

describe('SecondNav', () => {
  it('testSlot', () => {
    const wrapper = shallowMount(SecondNav, {
      slots: {
        default: () => 'test',
      },
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})
