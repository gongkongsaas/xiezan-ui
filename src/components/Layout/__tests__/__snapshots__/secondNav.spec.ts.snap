// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondNav testSlot 1`] = `
<div class="second-nav">
  <div class="second-nav-item">test</div>
  <div class="drop-down" style="display: none;"><i class="iconfont">î˜™</i></div>
</div>
`;
